2_intermediate/chapter10/solutions/img_avg.py

```diff
@@ -37,9 +41,12 @@
 
 # write code to create newimg here
 def solution1():
```

Comment:
@Citrus716 could you change this to not be a function since the students haven't learned them yet?